### PR TITLE
Fix mantainer to qa-c team is some tests

### PR DESCRIPTION
**Pull request by ilausuch**
- Related ticket: https://progress.opensuse.org/issues/136916

### CODE REVIEW

**Review by ricardobranco777 — APPROVED** [review]

File: tests/containers/container_diff.pm [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # SUSE's openQA tests
 #
-# Copyright 2020-2021 SUSE LLC
+# Copyright 2023 SUSE LLC
```
Comment (grisu48) [inline tests/containers/container_diff.pm:3]:
```suggestion
# Copyright 2020-2023 SUSE LLC
```

File: tests/containers/ecs_anywhere.pm [hunk 2]
```diff
@@ -1,14 +1,15 @@
 # SUSE's openQA tests
 #
-# Copyright 2022 SUSE LLC
+# Copyright 2023 SUSE LLC
```
Comment (grisu48) [inline tests/containers/ecs_anywhere.pm:3]:
```suggestion
# Copyright 2022-2023 SUSE LLC
```

File: tests/containers/run_container_in_k3s.pm [hunk 3]
```diff
@@ -1,12 +1,12 @@
 # SUSE's openQA tests
 #
-# Copyright 2022 SUSE LLC
+# Copyright 2023 SUSE LLC
```
Comment (grisu48) [inline tests/containers/run_container_in_k3s.pm:3]:
```suggestion
# Copyright 2022-2023 SUSE LLC
```

File: tests/containers/third_party_images.pm [hunk 4]
```diff
@@ -1,12 +1,12 @@
 # SUSE's openQA tests
 #
-# Copyright 2022 SUSE LLC
+# Copyright 2023 SUSE LLC
```
Comment (grisu48) [inline tests/containers/third_party_images.pm:3]:
```suggestion
# Copyright 2022-2023 SUSE LLC
```

File: tests/containers/update_host.pm [hunk 5]
```diff
@@ -1,14 +1,15 @@
 # SUSE's openQA tests
 #
-# Copyright 2022 SUSE LLC
+# Copyright 2023 SUSE LLC
```
Comment (grisu48) [inline tests/containers/update_host.pm:3]:
```suggestion
# Copyright 2022-2023 SUSE LLC
```

File: tests/containers/validate_btrfs.pm [hunk 6]
```diff
@@ -1,6 +1,6 @@
 # SUSE's openQA tests
 #
-# Copyright 2020-2021 SUSE LLC
+# Copyright 2023 SUSE LLC
```
Comment (grisu48) [inline tests/containers/validate_btrfs.pm:3]:
```suggestion
# Copyright 2020-2023 SUSE LLC
```

File: tests/containers/zypper_docker.pm [hunk 7]
```diff
@@ -1,6 +1,6 @@
 # SUSE's openQA tests
 #
-# Copyright 2018-2019 SUSE LLC
+# Copyright 2023 SUSE LLC
```
Comment (grisu48) [inline tests/containers/zypper_docker.pm:3]:
```suggestion
# Copyright 2018-2023 SUSE LLC
```

**Review by grisu48 — COMMENTED** [review]
Let's consistently use or date range or consistently not use date ranges.

I've added suggestions for places where date ranges have been replaced by single years but would be also OK in removing all date ranges.

**Review by grisu48 — APPROVED** [review]

**Pull request merged**